mempool/src/core_mempool/transaction_store.rs

```diff
@@ -553,9 +553,9 @@ impl TransactionStore {
         if let Some(txns) = self.transactions.get_mut(address) {
             if let Some(txn) = txns.get_mut(&txn_replay_protector) {
                 let sender_bucket = sender_bucket(address, self.num_sender_buckets);
-                let ready_for_quorum_store = !self.priority_index.contains(txn);
+                let ready_for_quorum_store = self.priority_index.insert(txn);
 
-                self.priority_index.insert(txn);
+                
```

Comment:
Thanks! Can you also remove the newlines 557 and 558 (so there is only a single empty line)

Reply:
done